test/Microsoft.AspNet.Mvc.Core.Test/DefaultActionSelectorTests.cs

```diff
@@ -860,7 +861,7 @@ private static RouteContext CreateRouteContext(string httpMethod)
             var routeData = new RouteData();
             routeData.Routers.Add(new Mock<IRouter>(MockBehavior.Strict).Object);
 
-            var httpContext = new Mock<HttpContext>(MockBehavior.Strict);
+            var httpContext = new Mock<HttpContext>(MockBehavior.Loose);
```

Comment:
It makes a big messy to the tests, I'm not sure how important this is, lets talk in person.